Library/Homebrew/cmd/update-report.rb

```diff
@@ -249,8 +249,21 @@ def updated?
   def migrate_tap_migration
     report[:D].each do |full_name|
       name = full_name.split("/").last
-      next unless (dir = HOMEBREW_CELLAR/name).exist? # skip if formula is not installed.
       next unless new_tap_name = tap.tap_migrations[name] # skip if formula is not in tap_migrations list.
```

Comment:
I know you've just moved around existing code, but I think it would be easier to follow (and also avoid an overly long line) if this was split into two lines. Assignments in conditions are sometimes surprising and a careless reader could think that's actually a check for equality. Here's what I'm suggesting:

```ruby
new_tap_name = tap.tap_migrations[name]
next if new_tap_name.nil? # skip if not in tap_migrations list.
```